data/lang/module-scoop/en.json

```diff
@@ -0,0 +1,190 @@
+{
+  "ACCEPTED_ARMS_DEALER": {
+    "description": "",
+    "message": "Great! Thank you."
+  },
+  "ACCEPTED_ILLEGAL_GOODS": {
+    "description": "",
+    "message": "Thanks. You won't regret it!"
+  },
+  "ACCEPTED_LEGAL_GOODS": {
+    "description": "",
+    "message": "Thank you. You know a good deal when you see one!"
+  },
+  "ACCEPTED_RESCUE": {
+    "description": "",
+    "message": "Thank you very much. The crew will appreciate that."
+  },
+  "ADTEXT_ARMS_DEALER": {
+    "description": "",
+    "message": "PICK UP drifting container."
+  },
+  "ADTEXT_ILLEGAL_GOODS": {
+    "description": "",
+    "message": "EASY MONEY: Valuable information for sale."
+  },
+  "ADTEXT_LEGAL_GOODS": {
+    "description": "",
+    "message": "INFORMATION FOR SALE: Knowing the location of a debris field with valuable cargo."
+  },
+  "ADTEXT_RESCUE": {
+    "description": "",
+    "message": "HELP NEEDED for a rescue operation."
+  },
+  "CLIENT": {
+    "description": "",
+    "message": "Client:"
+  },
+  "DANGER": {
+    "description": "The risk level",
+    "message": "Danger:"
+  },
+  "DEADLINE": {
+    "description": "Must be delivered by",
+    "message": "Deadline:"
+  },
+  "DENY_1": {
+    "description": "",
+    "message": "Sorry, I'm too busy to talk to you right now!"
+  },
+  "DENY_2": {
+    "description": "",
+    "message": "Oh, I'm sorry, I don't have time right now."
+  },
+  "DETONATORS": {
+    "description": "",
+    "message": "Detonators"
+  },
+  "FAILURE_MSG_ARMS_DEALER": {
+    "description": "",
+    "message": "You fool. The client's not gonna like this! Good luck, loser!"
+  },
+  "FAILURE_MSG_RESCUE": {
+    "description": "",
+    "message": "Your unreliability has caused great harm. Get out of my sight!"
+  },
+  "HOW_MUCH_TIME": {
+    "description": "",
+    "message": "How much time do I have?"
+  },
+  "HOW_MUCH_TIME_ARMS_DEALER": {
+    "description": "",
+    "message": "My business partner is in orbit around {star} to avoid detection. It would be great if you could reach him before {date}."
+  },
+  "HOW_MUCH_TIME_ILLEGAL_GOODS": {
+    "description": "",
+    "message": "Don't waste time. Only the early bird catches the worm!"
+  },
+  "HOW_MUCH_TIME_LEGAL_GOODS": {
+    "description": "",
+    "message": "I don't know. But if I were you, I would be quick!"
+  },
+  "HOW_MUCH_TIME_RESCUE": {
+    "description": "",
+    "message": "It would be desirable if you could reach the location before {date}."
+  },
+  "INTROTEXT_ARMS_DEALER": {
+    "description": "",
+    "message": "Hello Commander. My name is {client}. It would be great if you could help me out. An unreliable freighter captain dropped my cargo near {planet} and has disappeared since. Your task would be to pick up the cargo and deliver it to ship {shipid} in orbit around {star}. I would pay you {cash}."
```

Comment:
The arms dealer text should ideally mention the fact that this is only one ton of goods (e.g. with a form option similar to the Cargo missions) but this is a low-priority papercut.